lib/copy-sync/copy-sync.js

```diff
@@ -182,7 +182,11 @@ function checkStats (src, dest) {
 function checkPaths (src, dest) {
   const {srcStat, destStat} = checkStats(src, dest)
   if (destStat.ino && destStat.ino === srcStat.ino) {
-    throw new Error('Source and destination must not be the same.')
+    console.warn(
```

Comment:
Considering your comment on the fact that inodes can be the same when copying across volumes, then this is a totally valid use case, I would remove this warning and the if statement alltogether.

Reply:
This is just a hotfix to disable the Inode check completely and just log a warning. Not only when copying across volumes. Anyway, this pull-requests can be closed. If I understand it correctly, the issue will be fixed in #667 too.